lib/manager/components/transform/FeedTransformation.js

```diff
@@ -0,0 +1,214 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import React, {Component} from 'react'
+import {Button} from 'react-bootstrap'
+import Select from 'react-select'
+
+import {getGtfsSpec, getGtfsPlusSpec, isModuleEnabled} from '../../../common/util/config'
+import {getTransformationName} from '../../util/transform'
+import VersionSelectorDropdown from '../version/VersionSelectorDropdown'
+
+import type {
+  Feed,
+  FeedTransformation as FeedTransformationType,
+  ReactSelectOption
+} from '../../../types'
+
+type Props = {
+  feedSource: Feed,
+  index: number,
+  onChange: (any, number) => void,
+  onRemove: (number) => void,
+  transformation: FeedTransformationType
+}
+
+/**
+ * Component that renders fields for a single feed transformation
+ * (e.g., ReplaceFileFromStringTransformation).
+ */
+export default class FeedTransformation extends Component<Props, {csvData: ?string}> {
+  state = { csvData: null }
+
+  _getFieldsForType = (type: string) => {
+    const {feedSource, transformation} = this.props
+    // Get selected version (if applicable).
+    const version = feedSource.feedVersions &&
+      feedSource.feedVersions.find(v => v.id === transformation.sourceVersionId)
+    const fields = []
+    let index = 0
+    switch (type) {
+      case 'ReplaceFileFromVersionTransformation':
+        fields.push(
+          <VersionSelectorDropdown
```

Comment:
Add TODO to not show the current version of the feed?

Reply:
Do you mean the latest feed version for a feed source? No, I don't think we want this. If a user uploads one GTFS file, and then they want to reference this as the source version for all future uploads, we would want to show that version. "Current version" really has no meaning here -- this is just a way for a user to select any version uploaded for the feed source.